src/hiero_sdk_python/client/client.py

```
@@ -277,6 +312,156 @@ def set_default_max_query_payment(self, max_query_payment: Union[int, float, Dec
+
+        if grpc_deadline > self._request_timeout:
+            warnings.warn(
+                "grpc_deadline should be smaller than request_timeout. "
+                "This configuration may cause operations to fail unexpectedly.",
+                UserWarning,
+            )
+
+        self._grpc_deadline = float(grpc_deadline)
+        return self
+
+    def set_request_timeout(self, request_timeout: Union[int, float]) -> "Client":
```

Ruff (0.14.14)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)